middleware/responder/sse/responder.go

```diff
@@ -0,0 +1,69 @@
+package sse
+
+/*
+ * SPDX-FileCopyrightText: 2023 Siemens AG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Author: Michael Adler <michael.adler@siemens.com>
+ */
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/go-openapi/runtime"
+	"github.com/go-openapi/runtime/middleware"
+	"github.com/olebedev/emitter"
+	"github.com/siemens/wfx/middleware/logging"
+)
+
+// Responder sends a stream of events to the client.
```

Comment:
Technically, it responds (i.e. sends) stuff to the client, so it may read "sends back" or "responds" but that's a NOC (no comment)...

Reply:
Agreed, I have rephrased the commented and added more information.